websites/docs-smoke-test/src/releases/excerpt-rendering-test.mdx

```diff
@@ -11,15 +11,15 @@ topics:
 
 <Subtitle>Subtitle Text</Subtitle>
 
-A plain paragraph `with inline code` in it. And **bold** and *italic*.
```

Comment:
I'm wondering if you had to make these markdown changes for any specific reason. If these are "mandatory" changes because of some compatibility issue, we should keep that in mind in case some content in the docs repo needs to be updated

Reply:
Oh no that was just prettier I think...nothing I did on purpose. But it didn't change anything according to the VRTs

Reply:
Ok, I'm not sure to be honest if we should include this change. The smoke test is supposed to cover specific usage of mardown that is widely used in the docs repo. What I'd like to avoid is to switch to a more "pretty" markup and not test corner cases that in the future could bite us back.
What do you think?

Reply:
Yes, let's keep this PR clean...I will remove these changes 👍🏼